src/cow-react/modules/limitOrders/services/tradeFlow/index.ts

```diff
@@ -11,6 +11,11 @@ import { Web3Provider } from '@ethersproject/providers'
 import { AddAppDataToUploadQueueParams, AppDataInfo } from 'state/appData/types'
 import confirmPriceImpactWithoutFee from '@src/components/swap/confirmPriceImpactWithoutFee'
 import { LOW_RATE_THRESHOLD_PERCENT } from '@cow/modules/limitOrders/const/trade'
+import { swapFlowAnalytics } from '@cow/modules/swap/services/common/steps/analytics'
+import { logSwapFlow } from '@cow/modules/swap/services/utils/logger'
```

Comment:
We mustn't use files from swap module in the limit-orders module.
If something is common, then it must be located in the `common` or `trade` module

Reply:
I wonder if this is sth that eslint could help us with, define our own rules. Maybe like this we can reduce progressively the coupling of modules

Reply:
I haven't heard about eslint rules for that, but would be great to control it somehow